package.json

```diff
@@ -43,15 +48,15 @@
     "jest": "18.1.0",
     "json-loader": "0.5.4",
     "object-assign": "4.1.0",
+    "openzeppelin-test-helpers": "^0.5.0",
     "path-exists": "2.1.0",
     "postcss-loader": "1.0.0",
     "promise": "7.1.1",
     "react-dev-utils": "^0.4.2",
     "recursive-readdir": "2.1.0",
     "strip-ansi": "3.0.1",
     "style-loader": "0.13.1",
-    "truffle-contract": "^3.0.0",
-    "truffle-solidity-loader": "0.0.8",
+    "truffle": "^5.0.14",
```

Comment:
Not 5.1?

Reply:
Easy mate, we are already upgrading a major version! One step at a time :-P